decentraland/unity-explorer
# feat: LOD Group

## What does this PR change?

Introduces LODGroups

Some remaining [TODOs](https://github.com/decentraland/unity-explorer/issues/1568) for a separate PR



## How to test the changes?

1. Launch the explorer
2. Walk around and see how lods appear and dissapear
3. If any LOD seems to pop in, please let me know

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
